Design.md

```diff
@@ -0,0 +1,35 @@
+## Product Goals: 
+- Give users a good set of handrails so in their use of the UI, they don’t get themselves stuck.
+  - Provide a ‘happy path’ that the extension doesn’t deviate from. Make it easy for users to follow this happy path and difficult for them to deviate from it.
+- Help establish a good working rhythm with Git that will help them develop a useful mental model .
+  - Expose the most frequently used commands that cover the majority of daily git use.
+- Increase Usability of Git
+  - Shield user from repetitive actions; surface state-related information.
```

Comment:
> surface state-related information

For me that's the key information - what has changed and why. Blame is useful for this as is viewing the history of commits - for the particular file you're viewing.